server/methods/core/shipping.js

```diff
@@ -54,9 +70,11 @@ function createShipmentQuotes(cartId, shopId, rates) {
   return update;
 }
 
-
 /**
- * @summary if we have items in the cart, ensure that we only have shipping records for shops currently represented in the cart
+ * @name pruneShippingRecordsByShop
```

Comment:
Guessing we're not using the `@membeof` here because it's private?

Reply:
Correct. `@private` methods don't need `@memberof`. But they should still have `@summary` for people to read in the code itself.